src/views/partials/doc-resource-races.ejs

```diff
@@ -285,6 +288,48 @@
   </tbody>
 </table>
 
+<h3>GET api/races/{index}/ability-bonuses/</h3>
```

Comment:
This endpoint doesn't exist. Should it?

Reply:
That was one of my questions I left up in the PR description, along with a few other ones that I'm not sure the answers to, hoping for feedback / input 🙂 

> - where should this type live?
> - should it be in it's own file?
> - should it be under the Race section?
> - should resources of this type be available via their own endpoint?

Reply:
You've learned my terrible secret. My reading comprehension is crap.

Reply:
I think for now, it doesn't make sense for this to endpoint to exist. Especially with Graphql so close. However, there is a section on the docs for common use patterns which it makes sense for this to live for now.

Reply:
I agree, that sounds like the right thing to do. I'll tweak this branch a bit tonight. Thanks!